.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "editor.defaultFormatter": "ms-python.black-formatter",
+  "pylint.enabled": true,
+  "pylint.args": [
+    "--load-plugins",
+    "pylint_django"
+  ],
+}
```

Comment:
why is this here ?

Reply:
added the `settings.json` to the repo because it seems useful to have here
added `pylint_django` to stop vscode from underlining our `Model.objects` everywhere

Reply:
It doesn not underline my. 
So i think we should remove this. This is a local dev environment setting.

Reply:
did you pip install the requirements? my reasoning was that someone who clones the project shouldn't see errors. 

the formatting settings (the "black" formatter) should also be in source control btw... if they're not could you tell me how to add them

Reply:
i dont quite remember but it didn't give me that error for a very long time now

we have pre commit to handle the formatting locally and in our ci.